schedule_lessons/accounts/forms.py

```diff
@@ -0,0 +1,19 @@
+from allauth.account.forms import SignupForm
+from allauth.account.adapter import get_adapter
+from allauth.account.utils import setup_user_email
+
+class MyCustomSignupForm(SignupForm):
+
+    def save(self, request):
+        full_name = request.POST.get('user_name').split()
```

Comment:
if we want to use this `split` approach, I think we should rename this variable to clearly indicate that it is an array (list) and not a string. Right now, the name `full_name` makes me think it contains something like
```
ahmed sidd
```
but it actually contains something like
```
['ahmed', 'sidd']
```